conformance/01_pull_test.go

```diff
@@ -152,6 +153,21 @@ var test01Pull = func() {
 		})
 
 		g.Context("Teardown", func() {
+			deleteManifestFirst, _ := strconv.ParseBool(os.Getenv(envVarDeleteManifestBeforeBlobs))
```

Comment:
looking through this one more time - could we move this env var check to run only once at the bottom of setup.go? Check something like `runContentDiscoverySetup` , it is initialized as a global then set based on env

Reply:
done